src/components/DuosHeader.js

```diff
@@ -103,7 +103,8 @@ export const headerTabsConfig = [
     children: [
       { label: 'DAR Requests', link: '/chair_console' },
       { label: 'Datasets', link: '/dataset_catalog' },
-      { label: 'DAC Members', link: '/manage_dac' }
+      { label: 'DAC Members', link: '/manage_dac' },
+      { label: 'My DAC\'s Datasets', link: '/dac_datasets' }
```

Comment:
Stylistic question: what are your thoughts on switching quotes for this situation (e.g.
```suggestion
      { label: "My DAC's Datasets", link: '/dac_datasets' }
```